ifsbench/data/tests/test_fetchhandler.py

```diff
@@ -0,0 +1,126 @@
+# (C) Copyright 2020- ECMWF.
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+"""
+Tests for the FetchHandler class.
+"""
+
+from pathlib import Path
+
+import pytest
+
+from ifsbench.data import FetchHandler
+
+
+@pytest.mark.parametrize('source_url', ['file://home/some_file.tar.gz'])
+@pytest.mark.parametrize('target_path', [Path('subdir/target.tar.gz')])
+@pytest.mark.parametrize('force', [True, False])
+@pytest.mark.parametrize('ignore_errors', [True, False])
+def test_fetchhandler_serialise(source_url, target_path, force, ignore_errors):
```

Comment:
For unit test I'd prefer a name that includes the function(s) being tested, i.e. something with _dump_config